ui/src/services/NodeVolumesUtils.js

```diff
@@ -137,7 +137,7 @@ export const volumeGetError = (status) => {
 
 const getPVList = (state) => state?.app?.volumes?.pVList;
 const getPVCList = (state) => state?.app?.volumes?.pVCList;
-const getAlerts = (state) => state?.app?.monitoring?.alert;
+const getAlerts = (state) => state?.app?.alerts;
```

Comment:
Hello @alexis-ld ,

Sorry, we still miss one thing that I forgot last time. 
We actually have a unit test for `getVolumeListData` function. It returns the data used by Volume Table. 
We may want to update the `stateApp` that used by the unit test.

```javascript
const stateApp = {
  app: {
    nodes: {...},
    pods: {...},
    volumes: {...},
    monitoring: {...},
    // we may want to add this.
    alerts: { list : [] }, 
}
```

I would suggest you run the unit test `npm run test` to see if the new changes will break the test or not.
Thank you!